### ISSUE TITLE
[Help] Can not install without docker

### ISSUE THREAD

**Issue by linbojin** [ItzCrazyKns/Perplexica#59]
**Describe the bug**
I want to install without docker, because my working environment can only do this. I refer to the user manual and everything is done, but just after entering the query, "WebSocket is already in CLOSING or CLOSED state." is displayed.

```
nvm install 18.17.0
nvm use 18.17.0
cd Perplexica
npm i 
npm run build
npm run start

cd ui/  
npm i 
npm run build
npm run start
```

ui/.env
```
NEXT_PUBLIC_WS_URL=ws://xxxx/aaa/proxy/3001
NEXT_PUBLIC_API_URL=http://xxx/aaa/proxy/3001/api
```
and api url is working fine.

**Screenshots**
![image](https://github.com/ItzCrazyKns/Perplexica/assets/5894707/f10e0fbd-6f40-4dcb-a38c-f887584b0a10)

Can I get more logs about WebSocket?  There are no errors in the terminal currently. 

![image](https://github.com/ItzCrazyKns/Perplexica/assets/5894707/352c85f1-6c0e-4237-b928-8a543e52ff98)
![image](https://github.com/ItzCrazyKns/Perplexica/assets/5894707/8d2505f3-0cf3-4b08-8519-8d3884cbeff0)

**Comment by ItzCrazyKns** [comment]
Hi, can you try refreshing the page and see if it works. Make sure the console is open while you refresh the page. If the error message again pops up after refreshing the page (before sending a message) it indicates that either the embedding model could not be loaded or the chat model.

**Comment by linbojin** [comment]
![image](https://github.com/ItzCrazyKns/Perplexica/assets/5894707/96471f90-9c64-4e00-9134-9e91b6fd74d0)

**Comment by linbojin** [comment]
> Hi, can you try refreshing the page and see if it works. Make sure the console is open while you refresh the page. If the error message again pops up after refreshing the page (before sending a message) it indicates that either the embedding model could not be loaded or the chat model.

Can I get error logs about  "the embedding/chat model could not be loaded"

**Comment by ItzCrazyKns** [comment]
You're caught by an CORS error, unfortunately we only provide support for the installation of Perplexica and not for its deployment on some domain. I've pushed a new commit that should fix this problem, the HTTP requests that were being sent were having no `Content-Type` headers and that might be the reason CORS was blocking it. Try pulling the latest version of Perplexica from the main branch and it should fix the problem

**Comment by linbojin** [comment]
Yes, the CORS error is gone after pulling the latest version, Thx.
There are no errors after refreshing the page (before sending a message).
![image](https://github.com/ItzCrazyKns/Perplexica/assets/5894707/91d2b20b-0ade-4240-b757-a9e52b44ef58)

but after I send a query,  the “webSocket is already in CLOSING or CLOSED state.” error message again pops up .
![image](https://github.com/ItzCrazyKns/Perplexica/assets/5894707/de181be9-ecce-4c5b-952b-4c123fe03264)

**Comment by ItzCrazyKns** [comment]
That's weird. You can join our Discord server here https://discord.gg/Sud4rUfU, so we can communicate in a better way and we can help you resolve your issue

**Comment by ItzCrazyKns** [comment]
The WebSocket server shouldn't just close the socket. Its made to close the socket only if the selected model is invalid that to when connecting with it

**Comment by xywglzephyr** [comment]
i met similar situation.and my error like this.
![image](https://github.com/ItzCrazyKns/Perplexica/assets/124440920/37d1105a-2d70-4f88-a0ca-145e7d65f94b)
my homepage is like the upper，and my setting about chat model provider can't change.
![image](https://github.com/ItzCrazyKns/Perplexica/assets/124440920/7075cb81-4849-414a-90ec-2daf2e9e489c)
is there work way for me?

**Comment by CesarCalvoCobo** [comment]
> azure_openai_api_key"

Hi @linbojin , if I may , a related offtopic question : how did you manage to include the Azure Open AI Models in the list . Thanks in advance
